index.html

```diff
@@ -57,6 +57,260 @@ <h2>I'm a Software Developer</h2>
           </li>
         </ul>
       </div>
+    </main>
+    <section>
+      <div>
+        <div class="story">
+          <div class="story-title">
+            <h4>My Recent Works</h4>
+          </div>
+          <div class="card-top">
+            <img src="./pics/image1.svg" alt="image one" />
+          </div>
+          <div class="post-title">
+            <h4>Multi-Post Stories</h4>
+          </div>
+          <div class="card-supporting-text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <div class="languages">
+            <ul>
```

Comment:
- [ ] I think in your attempt on fixing the previous reviewers' requests you made some bugs because the first card in the works is now not at the top, but hidden behind others, I suggest you fix this to make it look like the template


| ![hidding](https://user-images.githubusercontent.com/102636489/227739415-82e4e2a3-ea4b-4d64-9116-7e4bd7d76767.png) | ![missing](https://user-images.githubusercontent.com/102636489/227739417-1fd4d507-73d3-44b8-b6da-7fd189deea7e.png) |
| --- | --- |

Reply:
Fixed :heavy_check_mark: 

![Screen Shot 2023-03-26 at 00 15 49](https://user-images.githubusercontent.com/13707896/227742472-36b49b4e-5ec9-4433-a12e-e382c42b1c0e.png)